src/chart/SunburstChart.tsx

```diff
@@ -218,10 +211,9 @@ export const SunburstChart = ({
           role="region"
         >
           <Surface width={width} height={height}>
-            {children}
             <Layer className={layerClass}>{sectors}</Layer>
+            {children}
           </Surface>
-          {renderTooltip()}
```

Comment:
Does being inside or outside the surface make differences? I guess this is why the foreign object is needed?

Reply:
Yes it does - that's why the `foreignObject` and `xmlns` are necessary.

I suspect that this was the original motivation to have tooltip be a special element rendered outside of the regular children flow. For some reason the original author decided against using `foreignObject` even though it was available since version 1 of each browser? Anyway.

Also, in SunburstChart now (and soon in other charts) - it depends on the order. Tooltip no longer automagically renders above everything, but it now respects the order in its children array. This might be a breaking change for some, or fix for others - I think it's something we should mention in 3.0 changelog as breaking to be on the safe side.

Reply:
So you could potentially hide the tooltip behind other elements? That doesn't seem ideal but makes sense I guess